photon-client/package-lock.json

```diff
@@ -19,7 +19,8 @@
         "three-full": "^28.0.2",
         "vue": "^2.6.12",
         "vue-axios": "^2.1.5",
-        "vue-native-websocket": "git+https://github.com/PhotonVision/vue-native-websocket.git#5189f29",        "vue-router": "^3.4.3",
+        "vue-native-websocket": "git+https://git@github.com/PhotonVision/vue-native-websocket.git#5189f29",
```

Comment:
I'll admit I'm not super familiar with the guts of npm - is the extra `git@` to enforce a better protocol, a la [this gist](https://gist.github.com/taoyuan/bfa3ff87e4b5611b5cbe)?

Reply:
honestly i have no idea what this is i think it was done automatically

Reply:
This was done to force the usage of a better protocol yes. Without it, CI would error out.